### DISCUSSION TITLE
How to pass RASPI_AUTH_ENABLED=false via docker environmet

### DISCUSSION THREAD

**Discussion by rugarci** [RaspAP/raspap-docker#41] [Q&A]
I would like to disable authentication as I already manage this in a reverse-proxy

I see this is supported (https://github.com/RaspAP/raspap-webgui/issues/280) but I have tried to add RASPI_AUTH_ENABLED=false to container environment and it doesn't work

Any idea?

**Comment by billz** [comment]
That env variable isn't currently supported. However, `env-setup.sh` could be modified to make this work.
